src/cli/commands/licenses.js

```diff
@@ -124,15 +125,14 @@ export const {run, setFlags, examples} = buildSubCommands('licenses', {
       byLicense.set(manifest.name, manifest);
     }
 
-    console.log(
+    reporter.log(
       'THE FOLLOWING SETS FORTH ATTRIBUTION NOTICES FOR THIRD PARTY SOFTWARE THAT MAY BE CONTAINED ' +
-        `IN PORTIONS OF THE ${String(manifest.name).toUpperCase().replace(/-/g, ' ')} PRODUCT.`,
+        `IN PORTIONS OF THE ${String(manifest.name).toUpperCase().replace(/-/g, ' ')} PRODUCT.` +
+        '\n',
     );
-    console.log();
 
     for (const [licenseText, manifests] of manifestsByLicense) {
-      console.log('-----');
-      console.log();
+      reporter.log('-----\n');
```

Comment:
I am not 100% sure but afaik @kittens had a few console.log instead of reporters for a reason.
I don't remember which.